x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -231,26 +270,7 @@ export class RemoteClusterForm extends Component {
     const formattedSeeds = seeds.map(seed => ({ label: seed }));
 
     return (
-      <EuiDescribedFormGroup
-        title={
-          <EuiTitle size="s">
-            <h2>
-              <FormattedMessage
-                id="xpack.remoteClusters.remoteClusterForm.sectionSeedsTitle"
-                defaultMessage="Seed nodes for cluster discovery"
-              />
-            </h2>
-          </EuiTitle>
-        }
-        description={
-          <FormattedMessage
-            id="xpack.remoteClusters.remoteClusterForm.sectionSeedsDescription1"
-            defaultMessage="A list of remote cluster nodes to query for the cluster state.
-              Specify multiple seed nodes so discovery doesn't fail if a node is unavailable."
```

Comment:
I feel like we lose a bit of helpful information here. Can we move some of this content into the help text below the "Seed nodes" input, changing it to: "An IP address or host name, followed by the transport port of the remote cluster. Specify multiple seed nodes so discovery doesn't fail if a node is unavailable."

![image](https://user-images.githubusercontent.com/1238659/76665098-d2149580-6543-11ea-9c66-a3ee31d3a165.png)

Reply:
Good point!